packages/components/molecules/f-skeleton-loader/src/components/skeletons/CarouselCardBackground.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <div class="c-skeleton-loader-carouselCardBackground c-skeleton-loader-bone">
+        <image-block />
+    </div>
+</template>
+
+<script>
+import ImageBlock from './Image.vue';
+
+export default {
+    name: 'CarouselCardBackground',
```

Comment:
No need to set `name` in the components as vue-cli will do this automatically.

Reply:
@DamianMullins thanks didn't know that.